### ISSUE TITLE
Shaka calls for /h when detaching

### ISSUE THREAD

**Issue by boredom2** [google/shaka-player#1569]
**Have you read the [FAQ](https://bit.ly/ShakaFAQ) and checked for duplicate open issues?**:
yes

**What version of Shaka Player are you using?**:
2.4.4

**Can you reproduce the issue with our latest release version?**:
yes

**Can you reproduce the issue with the latest code from `master`?**:
yes

**Are you using the demo app or your own custom app?**:
custom App

**If custom app, can you reproduce the issue using our demo app?**:
I cant reproduce the Setup there

**What browser and OS are you using?**:
any, especially critical on Safari 11.1 on Mac

**What are the manifest and license server URIs?**:
any

**What did you do?**
load a Video, use shaka.detach(), play an MP4 (ad), and re-attach Shaka

**What did you expect to happen?**
detach() works without Issues and I can swap to the MP4

**What actually happened?**
somehow, the Player makes a Call to "/h" (I observed that already earlier), which leads to various Errors in the Media Handlings, in Safari 11.1, it leads to a complete Crash (of our Player) and everything stops.

Is there any Reason for this Call to "/h" when detaching Shaka or do I miss something here?
I can send you a Link to our Dev Players, if you want, where it occurs always, when the Ad wants to start (i.e. Shaka gets temporarily removed).

**Comment by boredom2** [comment]
Hi,

as always, thanks for your in-depth Analysis.
Nevertheless, 

![unbenannt](https://user-images.githubusercontent.com/409857/44894363-f0c88800-ad21-11e8-80eb-c1cc53a09045.PNG)

this Call is initiated by "unknown" and of Type "Media" - I am very uncertain, what this means :)
You can see the Effect here:

https://arc-dev-cbr.nexx.cloud/lab/ads-vast.php

and of course, I dont expect that you debug our Application Code :)
I just have no Idea, how this can happen and it only occurs, when Shaka is involved. In Scenarios without Shaka (iOS, progressive-Play on older Browsers etc), this does not come up at all.

**Comment by boredom2** [comment]
I was able to remove this Behaviour, although the Reason is still unclear, it is gone now by disabling EventListeners on the Media Object while detaching. 
Thanks your Time though!

**Comment by joeyparrish** [comment]
Well, I'm glad you have a solution, but I'm sorry I couldn't be more help.
